pkg/shim/runsc/BUILD

```diff
@@ -31,6 +31,7 @@ go_library(
         "@com_github_containerd_cgroups//:go_default_library",
         "@com_github_containerd_cgroups//stats/v1:go_default_library",
         "@com_github_containerd_cgroups//v2:go_default_library",
+        "@com_github_containerd_cgroups//v2/stats:go_default_library",
```

Comment:
can we remove this?

it is not needed and can't be resolved.

having "@com_github_containerd_cgroups//v2:go_default_library" at L33 will be good enough